samcli/local/init/templates/cookiecutter-aws-sam-hello-powershell/{{cookiecutter.project_name}}/template.yaml

```diff
@@ -0,0 +1,33 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+    Sample SAM Template for {{ cookiecutter.project_name }}
+
+# More info about Globals: https://github.com/awslabs/serverless-application-model/blob/master/docs/globals.rst
+Globals:
+    Function:
+        Timeout: 60
+
+
+Resources:
+
+    PowerShellFunction:
+        Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
+        Properties:
+            CodeUri: ./artifacts/Function.zip
+            Handler: Function::Function.Bootstrap::ExecuteFunction
+            Runtime: dotnetcore2.1
+            MemorySize: 512
+            Environment: # More info about Env Vars: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#environment-object
+                Variables:
+                    PARAM1: VALUE
+
+Outputs:
```

Comment:
Is PowerShell not support API Gateway Proxy Integration mode? 

If it does, would you be able to update the template.yaml to so ``sam local start-api` would work? It's missing the Events and the Outputs section.

Reply:
It does - but as mentioned above the warm up time on the functions means it usually wouldn't be a use case that most people adopt, so I took it out. Happy to put it in if needed.